src/Arbitrary.hs

```diff
@@ -804,7 +807,9 @@ instance Aeson.ToJSON D3DN where
 -- * event log
 
 instance Arbitrary EventLog where
-    arbitrary = garbitrary
+    arbitrary = EventLog <$> arbWord <*> nonEmpty
+      where
+        nonEmpty = (:) <$> garbitrary <*> garbitrary
```

Comment:
> these two `garbitrary` should be `arbitrary` instead.

that doesn't work, since both generate separate arguments to `(:)`.  `Couldn't match type ‘[a0] -> [a0]’ with ‘[EventLogItem]’`.